proposals/0251-font-styles.md

```diff
@@ -28,30 +28,55 @@ The HMI should be able to provide a list of supported hyper text elements to SDL
 ```xml
 <struct name="TextField" since="1.0">
 :
-    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minvalue="1" maxvalue="100" since="6.x">    
+    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minsize="1" maxsize="100" since="6.x">
+        <description>A list of supported html tags without the < > characters.</description>
     </param>
 </struct>
 ```
 
 The HMI API equivalent should exclude the versioning attributes.
 
+The head unit would just send an array of html tags without the < > characters. For a list of current html tags see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Inline_text_semantics
+For example:
+```json
+"hyperTextSupported":[ "b", "u", "i" ]
+```
+
 Alternative: An enum can be added that lists all the hyper text elements.
 
-With this implementation, technically the HTML tags take up part of the 500 character count restriction placed in `TextFieldStruct`. So, the character count for `TextFieldStruct` would go from 500 to 1000 in the HMI API. This struct is used for pretty much any text field in RPCs like `Alert`, `ScrollableMessage`, `Show` and so on.
+With this implementation, technically the HTML tags take up part of the 500 character count restriction placed in `TextFieldStruct`. So, the character count for `TextFieldStruct` would go from 500 to 1000 in the HMI API. This struct is used for pretty much any text field in RPCs like `Alert`, `ScrollableMessage`, `Show` and so on. This adds some HMI complexity and would require the SDL mobile libraries to handle translating the 1000 characters in text fields to 500 characters on older headunits.
 ```xml
 <struct name="TextFieldStruct">
   .
   .
   .
   <param name="fieldText" type="String" maxlength="1000" mandatory="true">
     <description>The text itself.</description>
+    <history>
+      <param name="fieldText" type="String" maxlength="500" mandatory="true">
+    </history>
   </param>
   .
   .
   .
 </struct>
 ```
 
+And the "width" parameter in `TextField` needs to be updated to 1000 characters
+```xml
+<struct name="TextField" since="1.0">
+    .
+    .
+    .
+    <param name="width" type="Integer" minvalue="1" maxvalue="1000" mandatory="true">
+      <description>The number of characters in one row of this field.</description>
+      <history>
```

Comment:
After discussion, since the changes to the `history` parameter were not agreed to, this will need a separate proposal. With that said, the history tag *is required* for proper implementation, so please create a revision proposal to add the history tags. Also, the current history tags you have here are not correct because they need version number information.

Reply:
Hmm, you really think a history tag would lad to dissension? I only ask because there's already a bunch of PRs in the backlog already and only 6 can be in review at a time and usually take a couple weeks to get through

Reply:
I don't think it will, but unfortunately when the SDLC votes to accept with revisions, _only_ those revisions have been accepted. So any additional revisions need to be known and voted upon. It's not ideal, but we missed it in the initial review.

Reply:
Also looking back we did discuss it in your comment, So I don't think it's particularly new:  https://github.com/smartdevicelink/sdl_evolution/issues/828#issuecomment-549915034